pages/dkp/konvoy/2.2/supported-operating-systems/index.md

```diff
@@ -16,27 +16,27 @@ Konvoy supports the following base Operating Systems.
+| [RHEL 8.4][rhel_8_4]  | 4.18.0-305.el8.x86_64              | Yes            | Yes  | Yes        |                      | Yes         |
+| [Ubuntu 18.04 (Bionic Beaver)][ubuntu_18] | 5.4.0-1080-aws | Yes            |      |            |                      |             |
+| [Ubuntu 20.04 (Focal Fossa)][ubuntu_20] | 5.13.0-1031-aws  | Yes            |      |            |                      | Yes         |
+| [SLES 15 SP3][sles_15] | 5.3.18-150300.59.63-default       | Yes            |      | Yes        |                      | Yes         |
 | [Oracle Linux RHCK 7.9][RHCK] | kernel-3.10.0-1160.el7   | Yes            | Yes  |            |                      |             |
 
 ## Pre-Provisioned/On Premises
 
 <!-- vale Vale.Spelling = NO -->
-| Operating System      | Kernel                           | Default Config | FIPS | Air Gapped | FIPS with Air Gapped | GPU Support <!-- vale Vale.Spelling = YES --> |
-|-----------------------|----------------------------------|----------------|------|------------|----------------------|-------------|
-| [CentOS 7.9][centos7] | 3.10.0-1160.el7.x86_64           | Yes            | Yes  | Yes        |                      |             |
-| [RHEL 7.9][rhel_7_9]  | 3.10.0-1160.el7.x86_64           | Yes            |      |            |                      | Yes         |
-| [RHEL 8.4][rhel_8_4]  | 4.18.0-305.el8.x86_64            | Yes            |      |            |                      | Yes         |
-| [Flatcar][flatcar]    | 3033.2.x                         | Yes            |      |            |                      |             |
-| [SLES 15 SP3](https://documentation.suse.com/sles/15-SP3/)    |                                  | Yes            |      |            |                      |             |
-| [Oracle Linux RHCK 7.9][RHCK] | kernel-3.10.0-1160.el7   | Yes            |      |            |                      |             |
+| Operating System       | Kernel                           | Default Config | FIPS | Air Gapped | FIPS with Air Gapped | GPU Support <!-- vale Vale.Spelling = YES --> |
+|------------------------|----------------------------------|----------------|------|------------|----------------------|-------------|
+| [CentOS 7.9][centos7]  | 3.10.0-1160.el7.x86_64           | Yes            | Yes  | Yes        |                      |             |
+| [RHEL 7.9][rhel_7_9]   | 3.10.0-1160.el7.x86_64           | Yes            |      |            |                      | Yes         |
+| [RHEL 8.4][rhel_8_4]   | 4.18.0-305.el8.x86_64            | Yes            |      |            |                      | Yes         |
+| [Flatcar][flatcar]     | 3033.2.x                         | Yes            |      |            |                      |             |
+| [SLES 15 SP3][sles_15] | 5.3.18-150300.59.63-default      | Yes            |      |            |                      |             |
```

Comment:
Do we want to clarify this table by adding a KIB column? In the spreadsheet, it says that the 'default config' is not yet supported. And here we are saying it is. I think we need to somehow call out SLES 15 SP3 using KIB. 

https://docs.google.com/spreadsheets/d/1pIaseCIbDtN5hI7Qn05aQONMiSWjGTs2BWywzBkbYSk/edit#gid=228716742

Reply:
I think we should add a KIB column, but I think that's outside the scope of this PR.  The point here was just to add in the other Kernals.

Reply:
alright agree! can we open a JIRA?

Reply:
Done:
https://jira.d2iq.com/browse/D2IQ-90916